src/main/groovy/com/ullink/gradle/opencover/OpenCoverPlugin.groovy

```diff
@@ -23,6 +23,12 @@ class OpenCoverPlugin implements Plugin<Project> {
             def version = task.getOpenCoverVersion()
             downloadOpenCover(project, version)
         }
+
+        task.conventionMapping.map 'reportGeneratorVersion', {'2.5.11.0'}
```

Comment:
does this hardcode this exact version?

Reply:
I've seen it is similar to the OpenCover versioning, we can give a version for the report generator, but if none is provided this will be the default. Maybe @gluck  can confirm this.

Reply:
yes, convention mappings are values used if none is provided, I think it's good not to force users to set one.